megamek/src/megamek/common/actions/WeaponAttackAction.java

```diff
@@ -1503,9 +1505,14 @@ private static String toHitIsImpossible(IGame game, Entity ae, int attackerId, T
                 return Messages.getString("WeaponAttackAction.TooLowForFrontSide");
             }
             // Aft-mounted weapons can only be fired at targets at least 1 altitude lower
-            if ((weapon.getLocation() == Aero.LOC_AFT)) {
-                if (ae.isAirborne() && (altDif > -1)) {
+            // For grounded spheroids, weapons can only be fired at targets in occupied hexes, 
+            // but it's not actually possible for a unit to occupy the same hex as a grounded spheroid so
+            // we simplify the calculation a bit
+            if ((weapon.getLocation() == Aero.LOC_AFT) && (altDif > -1)) {
+                if (ae.isAirborne()) {
                     return Messages.getString("WeaponAttackAction.TooHighForAft");
+                } else {
+                    return Messages.getString("WeaponAttackAction.GroundedSpheroidDropshipAftWeaponRestriction");
```

Comment:
The rules for landing dropships (TW, p. 87) says that the central hex is reduced by one level if not paved/road/water and the surrounding hexes are reduced to the same level if they started higher. This means that a dropship could land overhanging a cliff and a unit could get into the same hex underneath. This should only prevent the attack for range > 0.

Reply:
That may be true, but currently, MegaMek does not implement overhanging dropship behavior - instead, the grounded spheroid "hugs" the ground if deployed over a cliff, and flattens the terrain out if it lands on uneven terrain regardless of terrain elevation differences.

That being said, I've introduced logic to check for the theoretical presence of another unit underneath one of the dropship's occupied hexes in case we do implement overhang behavior somehow.